src/schema/client.js

```diff
@@ -3,25 +3,12 @@
 // device
 import { blob, sqliteTable, text } from 'drizzle-orm/sqlite-core'
 import { dereferencedDocSchemas as schemas } from '@mapeo/schema'
-import { NAMESPACES } from '../core-manager/index.js'
 import { jsonSchemaToDrizzleColumns as toColumns } from './schema-to-drizzle.js'
 import { backlinkTable } from './utils.js'
 
 export const projectTable = sqliteTable('project', toColumns(schemas.project))
 export const projectBacklinkTable = backlinkTable(projectTable)
-export const projectKeysTable = generateProjectKeysTable()
-
-function generateProjectKeysTable() {
-  /** @type {Record<string, import('drizzle-orm/sqlite-core').SQLiteColumnBuilder>}*/
-  const columns = {
-    projectId: text('projectId').notNull().primaryKey(),
-    projectSecretKey: blob('projectSecretKey'),
-  }
-
-  for (const namespace of NAMESPACES) {
-    const columnName = namespace + 'EncryptionKey'
-    columns[columnName] = blob(columnName)
-  }
-
-  return sqliteTable('projectKeys', columns)
-}
+export const projectKeysTable = sqliteTable('projectKeys', {
+  projectId: text('projectId').notNull().primaryKey(),
+  encryptionKeys: blob('encryptionKeys').notNull(),
```

Comment:
Making this not nullable makes sense right?

Reply:
yep. There will always be at least a projectKey